src/librustdoc/html/static/rustdoc.css

```diff
@@ -116,6 +116,9 @@ h4:not(.method):not(.type):not(.tymethod):not(.associatedconstant) {
 	padding-bottom: 6px;
 }
 h1.fqn {
+	display: flex;
+	gap: 8px;
```

Comment:
This is not supported on safari or on IE 12, so can't be used.

Reply:
I can change the gap to a margin if the rest is OK.

Reply:
As long as it works on both iOS safari and IE12, fine by me! Also: please add screenshots. :)

Reply:
OK! Puzzled by IE12, though. Do you mean IE11 or Edge?

Reply:
We don't support IE11 anymore since very recently. So if IE12 is actually edge, well, go for edge support. But most of the time, the issue is simply safari on iOS being terribly bad.

Reply:
Replaced the flex gap with a margin, and added test screenshots below.